visualizations/frontend/filters/TTextInput.vue

```diff
@@ -271,7 +271,7 @@ export default {
       set(value) {
         this.text_internal = value;
         // Todo: Remove this when spa is live
-        this.setFilterValue("query", this.text_internal, true);
```

Comment:
The setFilterValue function used to accept 3 params, that was reduced to 2 but all of the instances that called it did not get updated. Since this update increased the number of params to 3, all of those old instances needed to be cleaned up.